CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # beeline-python changelog
 
+## 2.13.0
+
+### Features
+
+We have added new functionality for `http_trace_parse_hook` and `http_trace_propagation_hook`. These features allow beeline users to process incoming tracing-related headers, allowing for eventual interoperability between beelines, OpenTelemetry and AWS tracing.
```

Comment:
I don't think we should advertise the AWS header parsing / propagation until we review how we want it to work. We've come across a few scenarios we need to work through first.

```suggestion
We have added new functionality for `http_trace_parse_hook` and `http_trace_propagation_hook`. These features allow beeline users to process incoming tracing-related headers, allowing for eventual interoperability between Honeycomb, OpenTelemetry (W3C) and other tracing formats.
```

Reply:
Ah, yep, 👍 on getting rid fo AWS reference.